src/index.html

```diff
@@ -8,11 +8,24 @@
 </head>
 <body>
     <div id="controls" class="hidden">
-      <div id="file-selector">
+      <div id="options-panel">
+        <div>
+          <h3>Options</h3>
```

Comment:
:question: can we rename it to something more explicit? this is about load option

Reply:
I put a generic name in case we had other options (not just for the load).

Moreover, the fit option will also be used later to change the fit without the load. So I find it will be confusing.

Reply:
For fit without load, I see that as an additional action, so extra html elements will be needed (this will be probably managed with buttons or checkbox)
We can keep option here but rename Fit in fit on load if you prefer

Reply:
Done